Use environment variables in config files

tjcutajar opened the pull request:
Resolves #43

anuragkapur commented:
Cheers @tjcutajar ! I'll review this tonight..

anuragkapur commented on atam4j/src/test/java/me/atam/atam4j/configuration/ConfigLoaderTest.java:
```diff
@@ -7,9 +8,45 @@
         assertThat(dummyConfigurationConfigLoader.getTestConfig().getSomeDummyAttribute(), CoreMatchers.is("someValue"));
     }
 
+    @Test
```
I would prefer to have the tests use a single config file - `config-loader-test.yml` instead of a 2nd `config-loader.test-env.yml`. 
Reasons:
* Simpler with a single file to manage
* Both files currently using the same attribute `someDummyAttribute` which can be a source of confusion. Certainly confused me in the first look and then I looked again.

anuragkapur commented:
Thanks @tjcutajar looks great! Sorry for the delayed review; the release automation took longer than I expected.

In addition to the minor comment added above, could you also please update the README with info on this feature?

Cheers!

tjcutajar commented:
@anuragkapur I've addressed your comment. Yes the wiki should be the place for explaining things like this. There isn't a way to submit a PR for the wiki but you can merge it manually from here https://github.com/tjcutajar/atam4j/wiki/Using-Configuration-Files

anuragkapur commented:
Thanks. Wiki page: https://github.com/atam4j/atam4j/wiki/Using-Test-Config-Files
